freesurfer_post/interfaces/tabular.py

```diff
@@ -395,24 +395,49 @@ def _run_interface(self, runtime):
         output_dir = Path(self.inputs.output_dir) / subject_id
         output_dir.mkdir(parents=True, exist_ok=True)
         output_prefix = f'{subject_id}_{session_id}' if session_id else subject_id
-        out_tsv = output_dir / f'{output_prefix}_brainmeasures.tsv'
-        out_json = output_dir / f'{output_prefix}_brainmeasures.json'
+        atlas_tsv = output_dir / f'{output_prefix}_seg-Freesurfer_morph.tsv'
+        whole_brain_tsv = output_dir / f'{output_prefix}_desc-Freesurfer_qc.tsv'
+        atlas_json = output_dir / f'{output_prefix}_seg-Freesurfer_morph.json'
+        whole_brain_json = output_dir / f'{output_prefix}_desc-Freesurfer_qc.json'
 
         # Extract just the values from the audit data
         data_value = {key: value['value'] for key, value in fs_audit.items()}
         data_df = pd.DataFrame([data_value])
-        # Convert column names to snake case
-        data_df.columns = [
-            col.lower().replace('-', '_').replace('.', '_') for col in data_df.columns
-        ]
+
         # Rename subject_id to participant_id
         data_df = data_df.rename(columns={'subject_id': 'participant_id'})
-        # Reorder columns to have participant_id first
+
         cols = data_df.columns.tolist()
-        cols.remove('participant_id')
-        data_df = data_df[['participant_id'] + cols]
-        # Replace missing values with "n/a"
-        data_df = data_df.fillna('n/a')
+        id_cols = ['participant_id', 'session_id']
+        id_cols = [col for col in id_cols if col in cols]
+
+        # Convert column names to snake case
+        data_df.columns = [
+            col.lower().replace('-', '_').replace('.', '_') for col in cols
+        ]
+
+        # Split data_df into two dataframes, one for the atlas and one for the whole brain measures
+        suffixes = [
+            '_nvoxels',
+            '_volume_mm3',
```

Comment:
Rename `volume_mm3` to `volume` and include a `'Units': 'mm3'` field in the json.

Reply:
I like the mm3 because it's what freesurfer already has in the name, doesn't take up much space, and saves you a look up in a json file, so I'd vote for keeping it without as is